addons/base_import/static/src/scss/base_import.scss

```diff
@@ -22,10 +22,12 @@
     }
 
     .oe_import_box {
-        padding: 8px;
-        background: #F0EEEE;
+        flex: 0 0 250px;
+        padding: 10px 14px;
+        background: white;
```

Comment:
Some left-over classes that could be boostrapped.
(bg-white, overflow-auto, font-weight-normal, ...)